### PR TITLE
Fix accept_confirm in another window

### PR DESCRIPTION
**Pull request by shunichi**
Fixed issue:
1. Open another window
2. Switch to the window
3. Confirm modal can not be accepted

### CODE REVIEW

File: spec/integration/session_spec.rb [hunk 1]
```diff
@@ -642,6 +642,7 @@
 
         @session.within_window(another_popup) do
           expect(@session.html).to include('Test')
+          @session.execute_script('window.close()')
```
Comment (twalpole) [inline spec/integration/session_spec.rb]: `another_popup.close()` outside the within_window block would be better

File: spec/integration/session_spec.rb [hunk 2]
```diff
@@ -956,6 +956,26 @@
...
+        end
+
+        @session.within_window(popup) do
+          expect {
+            @session.accept_confirm do
+              @session.click_link('Open for match')
+            end
+            expect(@session).to have_xpath("//a[@id='open-match' and @confirmed='true']")
+          }.not_to raise_error
+          @session.execute_script('window.close()')
```
Comment (twalpole) [inline spec/integration/session_spec.rb]: `popup.close()` outside the within_window block would be better

File: spec/integration/session_spec.rb (hunk 1 above)
Comment (shunichi) [reply spec/integration/session_spec.rb]: fixed

File: spec/integration/session_spec.rb (hunk 2 above)
Comment (shunichi) [reply spec/integration/session_spec.rb]: fixed

**Comment by twalpole** [comment]
Looks good - Thanks!

**Pull request merged**